marklogic-data-hub-central/ui/src/components/zero-state-explorer/zero-state-explorer.module.scss

```diff
@@ -1,7 +1,7 @@
 .container {
     display: flex;
     flex-direction: column;
-    min-height: 100%;
+    min-height: 99%;
```

Comment:
This change was made on request as I saw following in windows in explore tile. I still see the scroll with this change:

<img width="1669" alt="image" src="https://user-images.githubusercontent.com/16124406/89352341-b469d900-d668-11ea-90b4-730e6cdf3475.png">

Reply:
Verified that adjusting the height solved this scrollbar being visible.